PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-sparkoncosmos/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins{
+    id "java"
+    id "org.jetbrains.kotlin.jvm"
+}
+
+intellij {
+    plugins = ['java','maven',dep_plugins]
+}
+
+dependencies {
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.9.10")
+    compileOnly project(path: ':azure-intellij-plugin-lib')
+    runtimeOnly project(path: ':azure-intellij-plugin-lib', configuration: 'instrumentedJar')
+    compileOnly project(path: ':azure-intellij-plugin-lib-java')
+    runtimeOnly project(path: ':azure-intellij-plugin-lib-java', configuration: 'instrumentedJar')
+    compileOnly project(path: ':azure-intellij-plugin-hdinsight-lib')
+    runtimeOnly project(path: ':azure-intellij-plugin-hdinsight-lib', configuration: 'instrumentedJar')
+    implementation 'com.microsoft.azure:azure-toolkit-ide-common-lib'
+    implementation 'com.microsoft.hdinsight:azure-toolkit-ide-cosmos-spark-lib'
+    implementation 'org.dom4j:dom4j:2.1.3'
```

Comment:
please only declare used dependencies, remove this if not used directly in this module.